File: TP2DPBO2526C1/Python/negara.py
from benua import Benua


class Negara(Benua):
    def __init__(self, nama_benua, luas_benua, zona_waktu_utama, nama_negara, bahasa_resmi, mata_uang):
        super().__init__(nama_benua, luas_benua, zona_waktu_utama)
        self.nama_negara = nama_negara
        self.bahasa_resmi = bahasa_resmi
        self.mata_uang = mata_uang

File: TP2DPBO2526C1/Java/Negara.java
class Negara extends Benua {
    protected String nama_negara;
    protected String bahasa_resmi;
    protected String mata_uang;

    public Negara(String nb, long lb, String zvu, String nn, String br, String mu) {
        super(nb, lb, zvu);
        this.nama_negara = nn;
        this.bahasa_resmi = br;
        this.mata_uang = mu;
    }
}
